README.md

```diff
@@ -1,5 +1,208 @@
-# api-registry-pipeline-examples
+# API specification governance
 
-Examples of pipelines that can be added to a repository containing API specifications.
-The purpose with the pipeline is to trigger the pipeline, based on approved changes in API specifications,
-pushing the updated API specifications to an API specification registry.
+**This repository provides a comprehensive showcase of how the SINETIQ Framework could be used to successful handle `governance of API specifications` in the complete life cycle chain.**
+
+The covered processes are:
+- API **creation** and **approval**; how to create and approve an API specification.
+- API **implementation**; how to implement an API specification within a Provider or Consumer component.
+- API **deployment**; how to deploy a provider, or consumer, component instance and its API instance(s).
+- API **compliancy** test and **verification**; how to verify and test a provider API instance API specification implementation.
+
+_Sinetiq Framework provides core capabilities that support development and governance methodologies that forms a robust integration ecosystem for small, medium and large enterprise organizations with a heterogenous system landscape._
+
+<p>&nbsp;</p>
+
+## Getting Started
+
+This section will give an in-depth explanation of components used, the `demonstration setup` and hands-on step-by-step `installation guide`.
+
+The demonstration showcase also ships with hands-on code `examples` and components, to run, inspect & modify.
+
+
+### Demonstration Setup
+
+The environment for this demonstration is managed through a client computer and components listed below.
+
+- **API specification registry**: The `Sinetiq API Specification Registry` consist of published and approved api specifications with its generated unique identifier.
```

Comment:
Suggestion `The Sinetiq API Specification Registry holds approved and published API specifications and maps them to a unique and immutable identifier`